README.md

```diff
@@ -250,9 +252,47 @@ logger = EventTracer::Prometheus.new(
 )
 ```
 However, doing so defeats the purpose of being clear on what metrics we want to track and can also result in a performance penalty. 
-To make the metrics registration less tedious, we recommend you to standardize your custom metrics name. 
+To make the metrics registration less tedious, we recommend you to standardize your custom metrics name. We also provide a helper module to register metrics
+```ruby
+EventTracer::Helpers::PrometheusHelper.register_prometheus_metrics(
+    registry,
+    metric_type,
+    metric_name
+    labels # the list of your allowed_tags and default_tags
+)
+```
 
-- For multi-processes app, we need to choose the `DirectFileStore` for Prometheus's data store. Read on [Data store](https://github.com/prometheus/client_ruby#data-stores) for more information.
+Now you can auto-register metrics like this
+```ruby
+module EventTracking
+  def self.included(base)
+    EventTracer::Helpers::PrometheusHelper.register_prometheus_metrics(
+      registry,
+      :counter,
+      base.name,
+      [:allowed_tag_1, allowed_tag_2] # list of allowed_tags and default_tags
+    ) 
+  end
+end
+
+class Web::Controllers::Users::Update
+  include EventTracking # this will auto register a counter for metric `Web::Controllers::Users::Update`
+end
+```
+
+Note that for gauge metric, we can also specify the aggregation strategy for multi-proccesses app, by default it's `:max`
+
+```
+EventTracer::Helpers::PrometheusHelper.register_prometheus_metrics(
+  registry,
+  :gauge,
+  'web.controller.action',
+  labels, 
+  aggregation: :min
+)
+```
```

Comment:
After thinking a bit more, I think this helper won't really help much. I initially thought we could include the `included` method, too, but it would be too opinionated. For now, I think we can remove this and use a opinionated utils in our repo instead. Sorry for all the trouble 🙇